yt/data_objects/static_output.py

```diff
@@ -1116,6 +1115,8 @@ def _assign_unit_system(self, unit_system):
         self.unit_registry.unit_system = self.unit_system
 
     def _create_unit_registry(self, unit_system):
+        from yt.units import dimensions as dimensions
```

Comment:
this could be just `from yt.units import dimensions`, right?

Reply:
hum, yeah. I don't know what happened here !